docker/jetpack/download-jetpack.sh

```diff
@@ -4,6 +4,6 @@ set -e
 echo "mkdir -p /tmp/${JETPACK_VERSION}/${DEVICE_ID}"
 mkdir -p /tmp/${JETPACK_VERSION}/${DEVICE_ID}
 
-echo "sdkmanager --cli downloadonly --user ${NV_USER} --logintype ${NV_LOGIN_TYPE} --product ${PRODUCT} --version ${JETPACK_VERSION} --targetos ${TARGET_OS} ${DEVICE_OPTION} ${DEVICE_ID} --flash skip --license ${ACCEPT_SDK_LICENCE} --downloadfolder /tmp/${JETPACK_VERSION}/${DEVICE_ID}"
-sdkmanager --cli downloadonly --user ${NV_USER} --logintype ${NV_LOGIN_TYPE} --product ${PRODUCT} --version ${JETPACK_VERSION} --targetos ${TARGET_OS} ${DEVICE_OPTION} ${DEVICE_ID} --flash skip --license ${ACCEPT_SDK_LICENCE} --downloadfolder /tmp/${JETPACK_VERSION}/${DEVICE_ID}
+echo "xvfb-run sdkmanager --cli downloadonly --user ${NV_USER} --logintype ${NV_LOGIN_TYPE} --product ${PRODUCT} --version ${JETPACK_VERSION} --targetos ${TARGET_OS} ${DEVICE_OPTION} ${DEVICE_ID} --flash skip --license ${ACCEPT_SDK_LICENCE} --downloadfolder /tmp/${JETPACK_VERSION}/${DEVICE_ID}"
```

Comment:
I think this should be in a variable. Then if it isn't set, the script works normally without requiring the app to be installed. `${XVFB}` or `${VIRTUAL_FRAME_BUFFER}` or ...

Reply:
You bring up a good point, I’m open to using a build variable but feel the root cause is more a consequence of how the cli functionality is implemented in the sdkmanager.

If the  intent is to run the sdkmanager without the UI as part of the build process, having xvfb as a dependency could make sense as it makes that intent explicit.  

The fact that the script works on X11 systems as-is feels more like a side-effect of the sdkmanager cli not being truly windowless.  Requiring xvfb calls attention to this and makes builds portable to most systems out of the box.

An alternate way to look at this is that we could support build servers in Azure DevOps without modification if we require xvfb :)